sdk/communication/azure-communication-chat/tests/_shared/test_communication_identifier_serializer.py

```diff
@@ -6,92 +6,86 @@
 import unittest
```

Comment:
(not related to your fixes)
is there a reason this uses unittest instead of pytest?

Reply:
I don't know why we use unittest instead of pytest. I need to look into this.